packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -2073,30 +2075,27 @@ export class ProjectService extends BaseService {
 
         const selectReferences = [
             indexColumn.reference,
-            ...(groupByColumns || []).map((col) => col.reference),
-            ...(valuesColumns || []).map((col) =>
-                groupByColumns && groupByColumns.length > 0
-                    ? `${col.reference}_${col.aggregation}`
-                    : `${col.reference}_${col.aggregation} as ${col.reference}`,
+            ...(groupByColumns || []).map((col) => `${q}${col.reference}${q}`),
+            ...(valuesColumns || []).map(
+                (col) => `${q}${col.reference}_${col.aggregation}${q}`,
             ),
         ];
 
         const pivotQuery = `SELECT ${selectReferences.join(
             ', ',
-        )}, dense_rank() over (order by ${
+        )}, dense_rank() over (order by ${q}${
             indexColumn.reference
-        }) as row_index, dense_rank() over (order by ${
+        }${q}) as ${q}row_index${q}, dense_rank() over (order by ${q}${
             groupByColumns?.[0]?.reference
-        }) as column_index FROM group_by_query`;
+        }${q}) as ${q}column_index${q} FROM group_by_query`;
 
         if (groupByColumns && groupByColumns.length > 0) {
             // Wrap the original query in a CTE
             let pivotedSql = `WITH original_query AS (${userSql}), group_by_query AS (${groupByQuery}), pivot_query AS (${pivotQuery})`;
 
-            pivotedSql += `\nSELECT * FROM pivot_query WHERE row_index < ${
+            pivotedSql += `\nSELECT * FROM pivot_query WHERE ${q}row_index${q} < ${
                 limit ?? 500
-            } and column_index < 10 order by row_index, column_index`;
-
+            } and ${q}column_index${q} < 10 order by ${q}row_index${q}, ${q}column_index${q}`;
```

Comment:
completely forgot, why did we choose `10`?

Reply:
It was just a default until we make it configurable!